languagetool-core/src/main/resources/org/languagetool/resource/spelling_global.txt

```diff
@@ -20889,4 +20889,13 @@ Tico Tico
 Tico Tico no Fubá
 Tico-Tico no Fubá
 Schott Music
-Georges Catoire
\ No newline at end of file
+Georges Catoire
```

Comment:
@jaumeortola I added some words to the global-spelling.txt - just FYI, please let me know if any changes are needed